app/scripts/containers/root.js

```diff
@@ -0,0 +1,27 @@
+import React, {Component, PropTypes} from 'react'
+import {Provider} from 'react-redux'
+import {Router} from 'react-router'
+
+window.React = React
```

Comment:
This should probably be guarded and only be done non production envs as I think this is only used to attach the react devtools